recipe/meta.yaml

```
@@ -1,49 +1,63 @@
 {% set name = "dbt-adapters" %}
-{% set version = "1.13.0" %}
+{% set version = "1.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/dbt_adapters-{{ version }}.tar.gz
-  sha256: 2e4e743d3613e7d72319edf6eb85ea2ca57133472e32ff0148c1325f21ff6d16
+  url: https://github.com/dbt-labs/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5c0f8e34e6531b9d8787a629b7e2b63cb6eb47cc3b5d1c373a05a82fcbc9fda7
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39 or py>312]
+  # dbt-common is not avaliable for the s390x platform
+  skip: True  # [s390x]
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - hatchling
     - pip
+    - wheel
+    - setuptools
   run:
-    - python >={{ python_min }}
-    - dbt-common >=1.13,<2.0
+    - python
+    - dbt-common >=1.11,<2.0
     - pytz >=2015.7
     - agate >=1.0,<2.0
-    - mashumaro >=3.9,<3.15
-    - protobuf >=5.0,<6.0
-    - typing_extensions >=4.0,<5.0
-    # for mashumaro[msgpack]
+    - mashumaro >=3.0,<4.0
+    # The msgpack pkg needs for "mashumaro[msgpack]>=3.0,<4.0"
+    # correct version is msgpack>=0.5.6
     - msgpack-python >=0.5.6
+    - protobuf >=3.0,<5.0
+    - typing-extensions >=4.0,<5.0
 
 test:
+  source_files:
+    - tests
   imports:
     - dbt.adapters
   commands:
     - pip check
+    - PYTHONPATH=$(pwd) pytest -vvv tests/  # [not win]
```

I'm no expert but the Intertubes suggest that 

```suggestion
    - python -m pytest -v tests
```

might so the same without the awkward environment variable (number of `-v`s is your choice).